tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -505,7 +506,12 @@ cc_library(
         ":backend_configs",
         ":buffer_allocations",
         ":cusolver_context",
+<<<<<<< HEAD
+        ":cudnn_conv_runner",
+        ":cudnn_batchnorm_runner",
+=======
         ":gpu_conv_runner",
+>>>>>>> c1fb8a63327b204fcdbcea9c5c5bbb579f84991b
```

Comment:
Merge conflict.

Reply:
Sorry about this...I resolved these in the subsequent commit.